pycaption/scc/__init__.py

```diff
@@ -195,7 +195,7 @@ def detect(self, content):
         else:
             return False
 
-    def read(self, content, lang='en-US', simulate_roll_up=False, offset=0):
+    def read(self, content, lang='en-US', simulate_roll_up=False, offset=0, src_fps=30.0):
```

Comment:
It seems that the src_fps parameter gets passed a lot between methods without being used, so maybe it would be a good idea to declare it as a class attribute and assign its value in init. That way you can use it directly anywhere you need as `self.fps`. Please do the same for the writer class to have consistency.